web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -40,6 +40,9 @@
 </template>
```

Comment:
On line 34 (I can't make an inline comment there), could you change `v-slot:loader` to `#loader`? Devs may copy-paste from this form in the future, and it'd be good to have consistent usage of syntax.